src/odb/src/db/dbBTerm.h

```diff
@@ -92,6 +92,7 @@ class _dbBTerm : public _dbObject
   dbId<_dbBTerm> _ground_pin;
   dbId<_dbBTerm> _supply_pin;
   std::uint32_t _sta_vertex_id;  // not saved
+  std::pair<Point, Point> _constraint_region;
```

Comment:
This is a schema change and requires backward compatibility code along with serialization code.

Reply:
how does this work with bumps?

Reply:
Do you mean the pins that PPL places at the top layer? I'm adding the lower left and the upper right points to indicate the box the pins can be placed.